src/orders/components/OrderFulfillLine/OrderFulfillLine.tsx

```diff
@@ -110,54 +103,50 @@ export const OrderFulfillLine: React.FC<OrderFulfillLineProps> = props => {
         </Typography>
       </TableCellAvatar>
       <TableCell className={classes.colSku}>{line.variant?.sku}</TableCell>
-      {isPreorder ? (
-        <TableCell className={classes.colQuantity} />
-      ) : (
-        <TableCell
-          className={classes.colQuantity}
-          key={warehouseStock?.id ?? "deletedVariant" + lineIndex}
-        >
-          <TextField
-            type="number"
-            inputProps={{
-              className: classNames(classes.quantityInnerInput, {
-                [classes.quantityInnerInputNoRemaining]: !line.variant
-                  ?.trackInventory,
-              }),
-              min: 0,
-              style: { textAlign: "right" },
-            }}
-            fullWidth
-            value={lineFormQuantity}
-            onChange={event =>
-              formsetChange(line.id, [
-                {
-                  quantity: parseInt(event.target.value, 10),
-                  warehouse: lineFormWarehouse,
-                },
-              ])
-            }
-            error={overfulfill}
-            variant="outlined"
-            InputProps={{
-              classes: {
-                ...(isStockExceeded &&
-                  !overfulfill && {
-                    notchedOutline: classes.warning,
-                  }),
+      <TableCell
+        className={classes.colQuantity}
+        key={warehouseStock?.id ?? "deletedVariant" + lineIndex}
+      >
+        <TextField
+          type="number"
+          inputProps={{
+            className: classNames(classes.quantityInnerInput, {
+              [classes.quantityInnerInputNoRemaining]: !line.variant
+                ?.trackInventory,
+            }),
+            min: 0,
+            style: { textAlign: "right" },
+          }}
+          fullWidth
+          value={lineFormQuantity}
+          onChange={event =>
+            formsetChange(line.id, [
+              {
+                quantity: parseInt(event.target.value, 10),
+                warehouse: lineFormWarehouse,
               },
-              endAdornment: (
-                <div className={classes.remainingQuantity}>
-                  / {line.quantityToFulfill}
-                </div>
-              ),
-            }}
-          />
-        </TableCell>
-      )}
+            ])
+          }
+          error={overfulfill}
+          variant="outlined"
+          InputProps={{
+            classes: {
+              ...(isStockExceeded &&
+                !overfulfill && {
+                  notchedOutline: classes.warning,
+                }),
```

Comment:
same here - is it readable? would be nice to "name" it by extracting to some field / function